roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ds-ingress-nginx-controller.yml.j2

```diff
@@ -127,15 +147,27 @@ spec:
             timeoutSeconds: 5
             successThreshold: 1
             failureThreshold: 3
-{% if ingress_nginx_webhook_enabled %}
+{% if ( ingress_nginx_webhook_enabled or ingress_nginx_opentelemetry_enabled ) %}
```

Comment:
HI  @dkhachyan 

Is the `(` and `)` required ? Should we remove it :-)

Reply:
done